iceoryx_examples/request_response/README.md

```diff
@@ -8,25 +8,26 @@ two consecutive fibonacci numbers and the server responds with the next number i
 the sequence.
 
 We provide three examples, the very basic typed and untyped examples
-and the most natural setup combining a Server with a Listener and a Client using a Waitset.
-Since you can find the general setup and functionality of the client and the server
-also in the Listener/Waitset example, we will focus now on this.
+and the most natural setup combining a server with a Listener and a client using
+a WaitSet. Since you can find the general setup and functionality of the client
+and the server also in the Listener/WaitSet example, we will focus now on this.
```

Comment:
What is meant, is that we focus on the listener/waitset example since it contains all the important things from the other examples as well. Maybe I should rephrase it...